### PR TITLE
Tailwind CSS suggestion _micro_css-responsive-design.md

### PR DESCRIPTION
**Pull request by thetrevoradams**
Adding suggestion based on CSS apprentice badge Explorer [micro-badge]

### CODE REVIEW

**Comment by tetondev** [comment]
I'm not married to having the suggestion where it is, so if you prefer it somewhere else, let me know

File: badges-active/css/_micro_css-responsive-design.md [hunk 1]
```diff
@@ -40,7 +40,9 @@ Choose one:
 
 2) Demonstrate how you've done all of #1 in your work or side-work
 
-3) Something else creative that proves you are totally legit in this topic
+3) Consider setting up a project using something like [Tailwind CSS](https://tailwindcss.com/) and explore how to implement layouts demonstrating #1.
```
Comment (marybeshaw) [inline badges-active/css/_micro_css-responsive-design.md:43]: I don't have a problem with adding this as an option, but I wanted to share my two cents about this micro badge.

I ended up just having a discussion with my manager on this micro badge.  I don't think it should be overdone... the CSS apprentice badge is hard enough as it is.  Maybe it just falls under the "something else creative," but I feel like my team does so much of this particular micro badge in our daily work that I could easily point to areas of the app that demonstrate these principles.

Comment (spacerockzero) [reply badges-active/css/_micro_css-responsive-design.md:43]: Yeah this does feel a bit heavy for an apprentice badge.

Comment (spacerockzero) [reply badges-active/css/_micro_css-responsive-design.md:43]: I have guidance for estimated time per level in the template files:
> For Apprentice, the microbadges and time required should reflect a "feet-wet" basic literacy in the subject. 3-6hrs) (For Journeyman, the microbadges and time required should reflect a wide proficiency in the subject. 8-16hrs

Comment (spacerockzero) [reply badges-active/css/_micro_css-responsive-design.md:43]: Sorry i didn't realize these comments were pending for 2 months here

**Comment by redbugz** [comment]
Is there something specific about tailwind, or should we encourage any of the frameworks from a list like  this:
https://github.com/troxler/awesome-css-frameworks

**Comment by jamischarles** [comment]
@redbugz Tailwind has the lion share of the attention right now. Personally I've really enjoyed using it. I think it's sensible as the default shoutout and the language of "something like tailwind" leaves it ambiguous enough for other options. 👋 hi Logan!

**Pull request merged**